datastore/kubernetes.go

```diff
@@ -282,6 +282,39 @@ func (s *DataStore) UpdateLease(lease *coordinationv1.Lease) (*coordinationv1.Le
 	return s.kubeClient.CoordinationV1().Leases(s.namespace).Update(context.TODO(), lease, metav1.UpdateOptions{})
 }
 
+func (s *DataStore) ClearDelinquentAndStaleStateIfVolumeIsDelinquent(volumeName string) (err error) {
+	defer func() {
+		err = errors.Wrapf(err, "failed to ClearDelinquentAndStaleStateIfVolumeIsDelinquent")
+	}()
+
+	lease, err := s.GetLeaseRO(volumeName)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+
+	holder := *lease.Spec.HolderIdentity
+	if holder == "" {
+		// Empty holder means not delinquent.
+		// Ref: IsRWXVolumeDelinquent() function
+		return nil
+	}
+	if !(lease.Spec.AcquireTime).IsZero() {
+		// Non Zero lease.Spec.AcquireTime means not delinquent.
+		// Ref: IsRWXVolumeDelinquent() function
+		return nil
+	}
+
+	oneSecondAfterZero := time.Time{}.Add(time.Second)
```

Comment:
As noted above, we could also have done this by clearing the holder identity, rather than using another special time.  But I can live with this.